.gitignore

```diff
@@ -4,6 +4,7 @@ dist/
 __pycache__/
 .venv
 .pytest_cache
+venv/.env
```

Comment:
not sure I understand why we have this. why not use `.venv`?

Reply:
Happened automatically, we removed it